newIDE/app/src/EventsSheet/ParameterRenderingService.js

```diff
@@ -50,6 +58,10 @@ const components = {
 
 const stringRenderers = {
   forceMultiplier: renderForceMultiplierString,
+  globalvar: renderForceGlobalVariable,
+  scenevar: renderForceSceneVariable,
+  objectvar: renderForceObjectVariable,
```

Comment:
Double check the name of your methods ;) render**Force**ObjectVariable

Reply:
it is already called render**Force**ObjectVariable :D which is consistent with renderForceMultiplierString .. do you want me to rename it?

Reply:
I mean, "force" is a mistake here. This parameter is not related to a force at all. This should be called `renderObjectVariable`.

Reply:
ooh xD I will remove it lol sorry about that.I dont know why in my mind force meant that we are forcing it to render differently or something